README.md

```diff
@@ -13,11 +13,16 @@ This project goal is replace CD++ with a more flexible and better performant imp
 
 ## Quick start
 ### Requirements
-* A C++14 compliant compiler. 
+* A C++14 compliant compiler.
 
 ### Install
 * The library is headers only. Then, it is enough to put the include directory in the path the compiler looks up for them.
 
+### Running cadmium in concurrent mode
+By default, cadmium runs single threaded. There is also a concurrent version behind a flag, but it requires Boost.Thread library and some changes need to be made when compiling:
```

Comment:
I wouldn't say these are "changes", it sounds too me like I should go edit the library to use it. 
I would say "additional requirements"